### PR TITLE
184/Use android build gradle plugin 1.1.0 and add simple junit test

### PR DESCRIPTION
**Pull request by friedger**
This PR adds a simple unit tests for EventDateComparator (and also improves the comparator) by using android gradle plugin 1.1.0

screen shot 2015-02-22 at 22 51 54

### CODE REVIEW

**Comment by tasomaniac** [comment]
Looks good. :+1: 

I have a question about findbugs you have just applied. Is it working? I am sure that if it was working, it would have found couple of bugs in the code. :)

File: app/build.gradle [hunk 1]
```diff
@@ -121,6 +121,8 @@ dependencies {
     //Others
     compile 'org.jsoup:jsoup:1.7.3'
     compile 'joda-time:joda-time:2.3'
+    compile 'com.google.code.findbugs:jsr305:2.0.1'
```
Comment (HenriqueRocha) [inline app/build.gradle]: Is this relevant since there is no task to actually perform the code quality?

Comment (friedger) [inline app/build.gradle]: this is needed as the android gradle plugin uses a version for test and somehow it gets confused without this line.

**Comment by friedger** [comment]
I haven't applied findbugs, it is just to make the android gradle plugin happy. LGTM

**Comment by HenriqueRocha** [comment]
Okay, I'll merge it. I've used it before without findbugs so it's kind of strange.
We can remove it later if we find a workaround.

**Pull request merged**